mailbox/plugin/spamassassin/src/main/java/org/apache/james/mailbox/spamassassin/SpamAssassin.java

```diff
@@ -32,16 +32,20 @@
 public class SpamAssassin {
 
     private final MetricFactory metricFactory;
-    private final SpamAssassinConfiguration spamAssassinConfiguration;
+    private SpamAssassinConfiguration spamAssassinConfiguration;
 
     @Inject
-    public SpamAssassin(MetricFactory metricFactory, SpamAssassinConfiguration spamAssassinConfiguration) {
+    public SpamAssassin(MetricFactory metricFactory) {
```

Comment:
I would not use injection here. What I did for Quota is that only the Listener is injected and creates required components itself